dashboard/app/controllers/xhr_proxy_controller.rb

```diff
@@ -36,6 +36,7 @@ class XhrProxyController < ApplicationController
     api.football-data.org
     api.foursquare.com
     api.fungenerators.com
+    api.funtranslations.com
```

Comment:
I might push back on this one - there seemed to be at least one 'translation' available here that we wouldn't want to support. Happy to discuss more and point to the one I saw in particular.